application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -187,8 +163,9 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
                     getKeyByValue(VC_APPLICATION_TO_ID, applicationId).orElse("an activity");
         }
 
-        handleSubcommand(event, voiceChannel, applicationId, maxUses.getAsInt(), maxAge.getAsInt(),
-                applicationName);
+        handleSubcommand(event, voiceChannel, applicationId,
+                event.getOption(MAX_USES_OPTION).getAsLong(),
+                event.getOption(MAX_AGE_OPTION).getAsLong(), applicationName);
```

Comment:
This code doesn't even work, besides that, still extremely disagree to usage of Optional.

Optional exists to "wrap" around null, so handling becomes "easier" (subjective)
So why for gods sake, use Optional if it's literally a value that we encourage. (in this context)

This means you need to use `optional.orElse(null)` which honestly, is really damn useless 
Adds code, for no reason. Not for readability, maintainability, just, because you can?

Completely disagree with that.

Also talked about why we shouldn't use Optional on your [other PR](https://github.com/Together-Java/TJ-Bot/pull/303#pullrequestreview-824980517)